### ISSUE TITLE
experiment_id: parent_activity_id and AMIP-style simulations

### ISSUE THREAD

**Issue by taylor13** [WCRP-CMIP/CMIP6_CVs#613]
AMIP simulations in CMIP6 require no specific parent, so parent_experiment_id is set to "no parent".  Of course, any simulation must be initialized and  @glevava has pointed out in an email that they chose to initialize from "an historical branch time."  That is reasonable, but they could have instead chosen to branch from the piControl (which would have been equally acceptable).  

The reason that the "parent" is not particularly important in these runs is that the adjustment time for the atmosphere is a few weeks.  With prescribed SSTs and sea ice and due to the system being "chaotic", the initial conditions are "forgotten" after a few weeks and so any reasonable initial conditions will lead to climatically indistinguishable realizations a few weeks later.  There will be some memory of the initial state of some somewhat longer lived surface conditions (e.g., soil moisture, snow cover, vegetation), so some care should be taken that these reflect conditions of ca. 1870 (when amip-hist runs are initialized).  And those analyzing the output should generally exclude the first few weeks from detailed analysis because the initial conditions do matter then.

The bottom line is that "no parent" should be specified for all amip-style runs in CMIP6.

**Comment by durack1** [comment]
@taylor13 there doesn't appear to be anything obvious required to deal with this issue, we currently have:


experiment_id | activity id | experiment | tier | sub experiment id | parent experiment id | required model components | additional allowed model components | start year | end year | min number yrs per sim | parent activity id | description
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
amip | CMIP | AMIP | 1 | none | no parent | AGCM | AER CHEM BGC | 1979 | 2014 | 36 | no parent | DECK: AMIP

Can this be closed?

**Comment by taylor13** [comment]
Oh sorry, @durack1 .  I just opened the issue to replace my email response to @glevava.  There is no action required on our part.

**Comment by durack1** [comment]
@taylor13 I had assumed as much. It's good to have the commentary here, however I will now close
